internal/services/containers/kubernetes_cluster_resource.go

```diff
@@ -598,6 +598,14 @@ func resourceKubernetesCluster() *pluginsdk.Resource {
 								},
 							},
 						},
+						"ip_families": {
+							Type:     pluginsdk.TypeList,
+							Optional: true,
+							ValidateFunc: validation.StringInSlice([]string{
+								"IPv4",
```

Comment:
Use constants, `containerservice.IPFamilyIPv4` and `containerservice.IPFamilyIPv6`